### ISSUE TITLE
Page without route is available

### ISSUE THREAD

**Issue by Ridd0** [saltyshiomix/ark#11]
Hey, I have a question - why routes are still available even if we comment `AuthController`? For example, I mean `auth/login`.

I guess it's related to the Nextjs framework. It seems to me that `AuthController` is unnecessary (it's never executing). I've tried made console.log from `AuthController -> showLogin` but it's not working.

Thanks!

**Comment by saltyshiomix** [comment]
Hi @Ridd0 , thank you for your reporting the issue!

This is a bug of `server/app.module.ts`:

```ts
export class AppModule implements NestModule {
  private handleAssets(consumer: MiddlewareConsumer): void {
    consumer
      .apply(NextMiddleware)
      .forRoutes({
        path: '_next*',
        method: RequestMethod.GET,
      });

    // this will be handle all pages in the `pages/*`, so NestJS routs won't work
    // consumer
    //   .apply(NextMiddleware)
    //   .forRoutes({
    //     path: '*',
    //     method: RequestMethod.GET,
    //   });
  }
}
```

**Comment by saltyshiomix** [comment]
Fixed `v6.10.0`, so you can see `console.log()` 👍

**Comment by Ridd0** [comment]
Thanks, great job 👍
